server/src/com/google/refine/Refine.java

```diff
@@ -516,11 +516,11 @@ private void openBrowserFallback() throws IOException {
         Runtime rt = Runtime.getRuntime();
 
         if (SystemUtils.IS_OS_WINDOWS) {
-            rt.exec("rundll32 url.dll,FileProtocolHandler " + uri);
+            rt.exec(new String[] { "rundll32 url.dll,FileProtocolHandler ", String.valueOf(uri) });
         } else if (SystemUtils.IS_OS_MAC_OSX) {
-            rt.exec("open " + uri);
+            rt.exec(new String[] { "open ", String.valueOf(uri) });
         } else if (SystemUtils.IS_OS_LINUX) {
-            rt.exec("xdg-open " + uri);
+            rt.exec(new String[] { "xdg-open ", String.valueOf(uri) });
```

Comment:
On Linux this seems to work as well, and looks a bit cleaner I guess:
```suggestion
            rt.exec(new String[] { "xdg-open", String.valueOf(uri) });
```